client/packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -20,7 +20,7 @@ import { useTranslation } from '@common/intl';
 import { useTableStore } from './context';
 
 const DataTableComponent = <T extends RecordWithId>({
-  key,
+  id,
```

Comment:
this change resulted in a bunch of files, and is also changed already in the feature branch. It's annoying having console errors though.. so begone!